.clang-format

```diff
@@ -0,0 +1,104 @@
+# OSARA: Open Source Accessibility for the REAPER Application
+# .clang-format
+# Author: James Teh <jamie@jantrid.net>
+# Copyright 2023 Leonard de Ruijter
+# License: GNU General Public License version 2.0
+
+# Style based on Google's C++ style guide.
+BasedOnStyle: Google
+# Offset for access modifiers (public, protected, private).
+AccessModifierOffset: 0
+# Align code after an open bracket with block indentation.
+AlignAfterOpenBracket: BlockIndent
+# Do not align consecutive assignments.
+AlignConsecutiveAssignments: None
+# Do not align consecutive bit fields.
+AlignConsecutiveBitFields: None
+# Do not align consecutive declarations.
+AlignConsecutiveDeclarations: None
+# Do not align consecutive macros.
+AlignConsecutiveMacros: None
+# Do not align escaped newlines.
+AlignEscapedNewlines: DontAlign
+# Do not align operands in expressions.
+AlignOperands: DontAlign
+# Do not align trailing comments.
+AlignTrailingComments: Never
+# Do not allow short blocks (e.g., if statements without braces) on a single line.
+AllowShortBlocksOnASingleLine: Never
+# Do not allow short case labels in switch statements on a single line.
+AllowShortCaseLabelsOnASingleLine: false
+# Do not allow short enums on a single line.
+AllowShortEnumsOnASingleLine: false
+# Only allow empty functions on a single line.
+AllowShortFunctionsOnASingleLine: Empty
+# Do not allow short if statements on a single line.
+AllowShortIfStatementsOnASingleLine: Never
+# Do not allow short loops on a single line.
+AllowShortLoopsOnASingleLine: false
+# Do not pack function call arguments together.
+# A function call’s arguments will either be all on the same line or will have one line each.
+BinPackArguments: false
+# Do not pack function parameters together.
+# A function declaration’s or function definition’s parameters will either all be on the same line
+# or will have one line each.
+BinPackParameters: false
+# Add a space after the colon in bit fields.
+BitFieldColonSpacing: After
+# Break before binary operators that are not assignments.
+# E.g. break before "+", but not before "+=".
+BreakBeforeBinaryOperators: NonAssignment
```

Comment:
Hmm. We're pretty inconsistent on this one. Sometimes, we put `&&` at the end of the line, but other times, we put it at the start. That's mostly (maybe entirely?) my fault; I think my perspective on this has changed over time. Do you have a preference personally?

Reply:
I think i preferp putting it at the start. For example, in:
```cpp
if (
    something
    && something2
//    && something3
)
```

You can easily comment out lines, even the last predicate of the check.